src/components/SlateEditor/slateRendering.js

```diff
@@ -72,6 +72,12 @@ export const renderMark = (props, editor, next) => {
       return <sup>{children}</sup>;
     case 'sub':
       return <sub>{children}</sub>;
+    case 'code':
```

Comment:
Kvifor er code under renderMark og ikkje renderBlock? Er dette kun for inline rendering?

Reply:
Gløm det. Ser jo på class-name at det er inline.

Reply:
Neida. Var berre eg som ikkje var oppmerksom.